features/conference/components/Conference.js

```diff
@@ -18,13 +27,25 @@ class Conference extends Component {
      */
     render() {
         return (
-            <ConferenceContainer>
+            <ConferenceContainer onPress = { this.onPress.bind(this) }>
                 <LargeVideo/>
-                <Toolbar navigator = { this.props.navigator }/>
-                <FilmStrip/>
+                <Toolbar
```

Comment:
I was thinking that if we remove it from the DOM we'll need to be sure to keep all internal states in order to re-init where we were and was also afraid that it would not be as reactive.